Sky Atlas/Sky Atlas.html

```diff
@@ -12499,16 +12523,17 @@ <h1>Sections & Primary Docs</h1>
         </tr>
         <tr>
           <td>
-            <dfn>A.3.3 - Near Term Treatment</dfn>
+            <dfn>A.3.3 - Near-Term Treatment</dfn>
```

Comment:
This is not hyphenated in the forum post. @aa-lex can confirm if hyphenation should be added or not.

```suggestion
            <dfn>A.3.3 - Near Term Treatment</dfn>
```

Reply:
We can hyphenate these ones, and I will create an issue in Notion to check if we should update some more.